src/client/datascience/jupyter/kernels/jupyterKernelSpec.ts

```diff
@@ -34,6 +35,8 @@ export class JupyterKernelSpec implements IJupyterKernelSpec {
         this.metadata = specModel.metadata;
         // tslint:disable-next-line: no-any
         this.env = specModel.env as any; // JSONObject, but should match
+        // tslint:disable-next-line: no-any
+        this.interrupt_mode = specModel.interrupt_mode as any;
```

Comment:
This is part of the spec. If interrupt_mode = 'message' we should send a message to the kernel to interrupt. Rust is one such kernel.
**Not fixing this, just exposing this information so we can use it when fixing other realted issues**